File: problema02/sol_python/problema02.py
int; trajes = 0
float; v_unitario = 0
float; subtotal = 0
float; v_total = 0
float; descuento = 0
int; porcentaje1 =20
int; porcentaje2 = 25
int; porcentaje3 = 40
int; porcentaje4 = 50

n_trajes = int (input("Escriba el numero de trajes a comprar"))

v_unitario = float (input("Escriba el valor unitario"))

subtotal = v_unitario * n_trajes

if n_trajes == 1:
    descuento = porcentaje1 * subtotal/100
    v_total = subtotal - descuento
else:
    if n_trajes == 2:
        descuento = porcentaje2 * subtotal/100
        v_total = subtotal - descuento
    else:
        if n_trajes == 3:
            descuento = porcentaje3 * subtotal/100
            v_total = subtotal - descuento
        else:
            if n_trajes >3:
                descuento = porcentaje4 *subtotal/100
                v_total = subtotal - descuento


print("Subtotal",subtotal)
print("Descuento",descuento)
print("Valor Total",v_total)

File: problema02/sol_java/problema02/src/problema02/Problema02.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package problema02;

import java.util.Locale;
import java.util.Scanner;

/**
 *
 * @author jooec
 */
public class Problema02 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);
        int n_trajes;
        double v_unitario = 0;
        double subtotal = 0;
        double v_total = 0;
        double descuento = 0;
        int porcentaje1 = 20;
        int porcentaje2 = 25;
        int porcentaje3 = 40;
        int porcentaje4 = 50;

        System.out.println("Ingrese el numero de trajes a comprar");
        n_trajes = entrada.nextInt();
        System.out.println("Ingrese el valor unitario de los trajes");
        v_unitario = entrada.nextDouble();

        subtotal = v_unitario * n_trajes;

        if (n_trajes == 1) {
            descuento = subtotal * porcentaje1 / 100;
            v_total = subtotal - descuento;
        } else {
            if (n_trajes == 2) {
                descuento = subtotal * porcentaje2 / 100;
                v_total = subtotal - descuento;
            } else {
                if (n_trajes == 3) {
                    descuento = subtotal * porcentaje3 / 100;
                    v_total = subtotal - descuento;
                } else {
                    if (n_trajes > 3) {
                        descuento = subtotal * porcentaje4 / 100;
                        v_total = subtotal - descuento;
                    }

                }
            }
        }
        System.out.printf("Subtotal: %.2f\n" + "Descuento: %.2f\n" 
                + "Valor total: %.2f", subtotal,
                                       descuento,
                                       v_total);
    }
}
